### PR TITLE
Add box defitinition for foreman 3.2

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
Please always copy from nightly, never from a stable release. There are boxes missing now.

**Comment by zjhuntin** [comment]
@ekohl okay, redone based on nightly.

**Review by ekohl — APPROVED** [review]
Looks correct. Shouldn't be merged until we actually branch and want to test RC1, otherwise nightly upgrade pipelines break.

File: vagrant/config/versions.yaml [hunk 1]
```diff
@@ -104,6 +104,20 @@ installers:
...
 
+  - foreman: '3.2'
+    katello: '4.4'
+    pulpcore: '3.16'
+    puppet: 7
+    boxes:
+      - 'almalinux8'
+      - 'centos7'
+      - 'centos7-fips'
+      - 'centos8'
```
[the remark this answers is not in the document]
Comment (zjhuntin) [reply vagrant/config/versions.yaml]: Updated and removed centos8 :)

**Review by ekohl — APPROVED** [review]

**Pull request merged**